modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/BaseServerComponent.java

```diff
@@ -143,6 +144,12 @@ public AvailabilityType getAvailability() {
                 previousAvailabilityType = DOWN;
                 availabilityType = DOWN;
                 securityRealmReady = false;
+            } catch (UnauthorizedException e) {
+                previousAvailabilityType = DOWN;
+                availabilityType = DOWN;
+                throw new InvalidPluginConfigurationException("User authenticated via HTTP management interface does"
```

Comment:
I followed ```securityRealmReady``` approach first, but then I decided do to with rethrowing. Catch block rethrowing IPCE is needed anyway, because ```ASConnection.execute()``` throws it  when credentials are incorrect. I'll fix the code as you suggest and avoid rethrowing